### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by jmeloranta** [bilde2910/Hauk#149]
Hi,
I would like to use hauk to share location data of my family members. Since group shares can easily disappear when location sharing is stopped (the life time is set to 365 days), I frequently have to create a new one. With this the pin mechanism this is quite difficult in practice as I have to go over everyone's phone and put in the new pin. There is already username / password authentication, so it would be handy to be able to skip the pin system when logged in. Also, is there a way to ensure that a given group share would be permanent? Perhaps one could have permanent group shares specified in the configuration file that specified people can join? I have tried reserved links but what I gather is that this just makes sure that certain people can create this group share?

Thanks,
Jussi

**Comment by natrius** [comment]
You want a constant location-share? Is this https://github.com/bilde2910/Hauk/issues/130 what you mean? 
There are multiple open-source-apps that already achieve that, you might try https://owntracks.org/ for example.

**Comment by jmeloranta** [comment]
Yes, but hauk is nice and simple. It should be difficult to provide a configuration option to bypass pin?

**Comment by bilde2910** [comment]
Hauk is intended as a short-term location sharing service, and long-term sharing is really not in scope of the project. This is also the reason why Memcached/Redis is used as the storage backend for Hauk rather than a traditional DBMS such as MariaDB or PostgreSQL. Feature request for long-term storage is tracked in #130. As for having to enter a new PIN to join the group share, #121 will address this. PINs are a requirement because they are used to uniquely identify the specific group share that members will join.
